homeassistant/components/command_line/switch.py

```diff
@@ -76,7 +86,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     if not switches:
         _LOGGER.error("No switches added")
-        return False
+        return
```

Comment:
Unless I am mistaken, `setup_platform` shouldn't be returning a value.

Reply:
Its called here and its never checked so `None` is correct https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity_platform.py#L156